common/web/types/build.sh

```diff
@@ -93,7 +93,7 @@ function do_test() {
 
 #-------------------------------------------------------------------------------------------------------------------
 
-builder_run_action clean      rm -rf ./build/ ./tsconfig.tsbuildinfo
+builder_run_action clean      rm -rf ./build/ ./tsconfig.tsbuildinfo ./src/schemas/ ./node_modules/ ./obj/
```

Comment:
This is not related but not worth splitting into a separate PR IMO. Just means clean really does a proper clean of all built files in common/web/types. Doesn't impact CI really.